### PR TITLE
Update sexual identity question

### PR DESCRIPTION
**Pull request by MIslam92**
### What is the context of this PR?

- content updated and pipe name into question

- updated tests functional and integration

Card: https://trello.com/c/QmxMUypz/1490-update-sex-identity

### How to review
 
The usual

### CODE REVIEW

File: data/en/census_household.json [hunk 1]
```diff
@@ -2605,39 +2605,11 @@
                     "id": "sexual-identity",
                     "questions": [{
                         "id": "sexual-identity-question",
-                        "title": "Which of the following options best describes how you think of yourself?",
-                        "description": "This question is voluntary",
+                        "title": "Which of the following options best describes how <em>{{[answers.first_name[group_instance]] | format_household_name }}</em> thinks of themselves?",
```
Comment (shahi645) [inline data/en/census_household.json:2608]: Doesn't match the question in design exactly

**Review by shahi645 — CHANGES_REQUESTED** [review]

**Review by charlotte-hirst — APPROVED** [review]
Although Shahi is right it doesnt match the design, I think it sounds better and for now is consistent with other questions where we use the phrase 'best describes'.

**Review by shahi645 — APPROVED** [review]
squishy-squashy

**Comment by shahi645** [comment]
@jonnyshaw89 to look at and merge when the travis build passes 😉

**Pull request merged**